demos/list.html

```diff
@@ -1329,6 +1329,48 @@ <h3>Example - Interactive List (Dark Theme)</h3>
             evt.preventDefault();
           });
         }
+
+        var deactivateRippleHandler = null;
+        var checkboxListRippleSurfaceClass = 'checkbox-list-ripple-surface';
+
+        function clickHandler(evt, rippleFoundation, li) {
+          var checkbox = li.querySelector('.mdc-checkbox__native-control');
+          var isLabel = Array.prototype.some.call(checkbox.labels, function(label) { return label === evt.target; });
```

Comment:
Nit: Although `Array.prototype.some.call(...)` is technically The Right Way to do it, I tend to prefer:

`[].some.call(...)`

It's terser and (IMO) easier to read.

And unless you're writing performance-critical code, the cost of allocating a new object is negligible.

Reply:
definitely easier to read...changing